packages/ffmpeg/server/FFmpegExporterServer.ts

```diff
@@ -141,6 +148,23 @@ export class FFmpegExporterServer {
     }
   }
 
+  public async checkForAudioStream(asset: MediaAsset): Promise<boolean> {
+    return new Promise((resolve, reject) => {
+      ffmpeg.ffprobe(asset.src, (err, metadata) => {
+        if (err) {
+          console.error('error checking for audioStream for asset', asset.key);
```

Comment:
you can put the error into the console.error
```
console.error("error message", err);
```

Reply:
done